cfgov/cfgov/settings/local.py

```diff
@@ -40,7 +40,10 @@
 if os.environ.get('ENABLE_DEBUG_TOOLBAR'):
     INSTALLED_APPS += ('debug_toolbar',)
 
-    INTERNAL_IPS = ('127.0.0.1', '10.0.2.2')
+    # Local running instances: 127.0.0.1
+    # VirtualBox docker-machine instances: 10.0.2.2
+    # Docker for Mac instances: 172.18.0.1
+    INTERNAL_IPS = ('127.0.0.1', '10.0.2.2', '172.18.0.1')
```

Comment:
Is 172.18.0.1 guaranteed to be the host IP? See the [docs](https://docs.docker.com/docker-for-mac/networking/#use-cases-and-workarounds):

> The host has a changing IP address (or none if you have no network access). From 18.03 onwards our recommendation is to connect to the special DNS name host.docker.internal, which resolves to the internal IP address used by the host.

Reply:
That's a good point, although neither `host.docker.internal` nor `gateway.docker.internal` point to the correct IP. 

It seems like the correct IP is listed by `docker inspect cfgov-refresh_python_1` under NetworkSettings/Networks/cfgov-refresh_default/Gateway.

Reply:
Resolving this conversation. See comment below.